couchdb/couchdb.rb

```diff
@@ -81,7 +81,7 @@ def parent_interrupt
   ['--apihost',   '-a', GetoptLong::REQUIRED_ARGUMENT]
```

Comment:
Thanks for pointing this out Rost. It should be addressed now

On Thu, May 26, 2016 at 6:51 PM, Gagandeep singh <notifications@github.com>
wrote:

> In couchdb/couchdb.rb
> <https://github.com/CopperEgg/metrics-plugins/pull/16#discussion_r64743276>
> :
>
> > @@ -81,7 +81,7 @@ def parent_interrupt
> >    ['--apihost',   '-a', GetoptLong::REQUIRED_ARGUMENT]
>
> used Rubocop gem for analysis. Shows 376 warnings.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/CopperEgg/metrics-plugins/pull/16/files/3f3811c7d364eb4ecef1a3a80af92fe719ccda9d#r64743276>
>



-- 
Warm Regards,
Shankar S
(Technical Program Manager, Idera)